torch/__init__.py

```diff
@@ -22,7 +22,9 @@
 import sys
 import textwrap
 import threading
-from typing import Any, Callable, Dict, Optional, Set, Tuple, Type, TYPE_CHECKING, Union
+import typing
+from typing import Any, Callable, Dict, Optional, Set, Tuple, TYPE_CHECKING, Union
```

Comment:
Why not use this one?
```suggestion
from typing import Any, Callable, Dict, Optional, Set, Tuple, Type as _Type, TYPE_CHECKING, Union
```

Reply:
Thanks for the suggestions. Updated.